src/app/index.tsx

```diff
@@ -101,16 +102,23 @@ export default (options: AppOptions) => {
     ...hooks.map((hook) => hook(services)),
   ];
 
+  const enhancers: StoreEnhancer[] = [];
+
   if (Sentry.shouldCollectErrors) {
-    middleware.push(Sentry.initializeWithMiddleware());
+    enhancers.push(Sentry.createReduxEnhancer());
   }
 
   const store = createStore({
+    enhancers,
     initialState,
     middleware,
     reducer,
   });
 
+  if (Sentry.shouldCollectErrors) {
+    Sentry.initialize(store);
+  }
+
```

Comment:
It would be nice if I didn't need both `if`s here, we could alternatively leave the initializer as a no-op middleware and push it in the same spot, if that's preferred: https://github.com/openstax/rex-web/pull/1613/commits/965785d87989192bff5b5b9d571ebb78df46d5a3#diff-8a43619f80571a2033826efebd666b14d7d578f88af0f252421fe9266face636L45